### DISCUSSION TITLE
The CMD console appears for a moment

### DISCUSSION THREAD

**Discussion by Geopupkin** [brentvollebregt/auto-py-to-exe#474] [Q&A]
Hello everyone, please help me to solve the issue with the instant appearance of the CMD console during the exe startup process. This happens during the procedure of opening a folder by a specific path. This appears on this piece of code: 
`os.system(f' explorer "{target_path}"')`

**Comment by brentvollebregt** [comment]
Have you read the help post? If this is what happens when you double click on your script, then this is what will happen to it as an exe.

**Comment by Geopupkin** [comment]
Hi, thanks for the prompt response. Yes, I read different answers with a similar problem, but I could not solve it, I could only localize it by setting input() to different parts of the code. When running the script from pycharm, the cmd window does not appear and the script works as needed without displaying unnecessary messages. When compiling to exe and running code from exe, the code works fine, but the cmd window appears for a moment. понедельник, 25 марта 2024г., 10:06 +03:00 от Brent Vollebregt  ***@***.*** :

**Reply by brentvollebregt** [reply]
1. When double-clicking your .py script, does it function the way you expect?
2. When following the debugging steps, were there any errors you saw?

**Reply by Geopupkin** [reply]
1. Yes it function the way i expect/
2. The error (the appearance of the CMD console for a moment) appears on this line `os.system(f' explorer "{target_path}"')` Its meaning is that there is an appeal to the Windows operating system explorer in order to open the folder along the `target_path` path
3. I can provide all the code if needed.

**Reply by brentvollebregt** [reply]
Sorry, I'm confused, why is the cmd appearing for a moment? Following the debugging steps, the application should be run from cmd to make sure this doesn't happen.

**Reply by Geopupkin** [reply]
By "debugging" I thought you meant to find in the code the piece that causes CMD to appear. Through the substitution of "input()" and compilation from time to time in the exe, I found this piece of code. And I don't understand how to debug via CMD... Please help me figure it out..

**Reply by brentvollebregt** [reply]
There's a help post linked at the top of the UI, the bottom of the UI after every pack and in the readme under "Issues Using the Tool" - I imagine you would have seen this before creating this discussion. [In this post, there is a debugging section](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#debugging) that describes how you can find the actual error you are seeing. When following these steps, what did you discover?

> Please help me figure it out..

I provide many resources, the biggest being the debugging section in the help post. Please help me help you by reading the existing resources I provide.
